app/views/items/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1> Customer Details:</h1>
+
+<% @items.each do |item| %>
+    <li>Name: <%= item.name %></li>
+    <li>Address: <%= item.price %></li>
+    <li>City: <%= item.supermarket.name %></li>
+    <li>Purchase count: <%= item.customer_items.count %></li>
```

Comment:
Even though it is a very simple calculation, per MVC conventions, we don't want to do any AR calculations in the view. The view can instead call a model method (maybe something like `item.customer_count`) and then that model method in the Item model can look like this:
```
def customer_count
   self.customers.count
end
```

When we add a model method, we are making sure this data querying/calculation lives closer to the database, and we are then able to test it in a unit test (item_spec)